lib/tpm2_eventlog_yaml.c

```diff
@@ -413,85 +440,93 @@ bool yaml_uefi_image_load(UEFI_IMAGE_LOAD_EVENT *data, size_t size) {
     free(buf);
     return true;
 }
+#define EVENT_BUF_MAX BYTES_TO_HEX_STRING_SIZE(1024)
```

Comment:
Is this used?

Reply:
This macro was previously defined further down, and I just moved it up because the function `yaml_gpt()` also uses this macro.